script/Ciel/Day 10/10_01_CIEL10_4_2.txt

```diff
@@ -137,8 +137,8 @@ If used properly, it can single-handedly isolate a Dead Apostle from their troop
 [sha:f0c11f80d6fdf83772837cb76f77edd0168794fe]{
 -- Page 196, Offset 27584.
 -- 結界内に閉じこめるには対象を<黒鍵|こっけん>で囲む必要がありますし、大聖堂のような大魔術は連発できないんです。
-To successfully trap the target within the barrier you need to first surround them with the Black Keys,
- and Grand Magecraft like the Cathedral can't be cast in quick succession.
+You need to surround a target with Black Keys before you can use it to trap them,
+ and since the Cathedral is a Grand magecraft it can't be used repeatedly in quick succession.
```

Comment:
Seems to have some sloppy editing? Grand Magecraft can be used in its uncountable form.
>and since the Cathedral is Grand Magecraft, it can't be used in quick succession.